### PR TITLE
feat: introduce auto suggest for broken backlinks

### PR DESCRIPTION
**Pull request by dzehnder**
This PR implements a first iteration of https://jira.corp.adobe.com/browse/SITES-22462 (auto suggest broken backlinks),  based on organic keywords, as suggested [by research](https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=AEMSites&title=SEO+%7C+Broken+Backlinks+Auto-Suggest)

### CODE REVIEW

**Review by solaris007 — APPROVED** [review]

File: .nycrc.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "text"
   ],
   "check-coverage": true,
-  "lines": 100,
-  "branches": 100,
-  "statements": 100
+  "lines": 90,
+  "branches": 90,
+  "statements": 90
```
[the remark this answers is not in the document]
Comment (dzehnder) [reply .nycrc.json:9]: of course ❤️ 
this was only for testing purposes 😄

**Review by iuliag — APPROVED** [review]
Looks good as a first version of auto-suggest 👍🏻
It would be good to have the documentation of the approach also in the wiki or in a readme, so we can point to/update it more easily.

**Review by iuliag — COMMENTED** [review]
Some adjustments are needed so that the matching code logic works with top keyword coming from top pages

File: src/support/utils.js [hunk 2]
```diff
@@ -142,3 +143,152 @@ export function getUrlWithoutPath(url) {
...
+};
+
+/**
+ * TODO: remove again, copied from importer
+ * @param s3Client
+ * @param config
+ * @param context
+ * @returns {Promise<any|*[]>}
+ */
+export async function getStoredMetrics(s3Client, config, context) {
```
[the remark this answers is not in the document]
Comment (dzehnder) [reply src/support/utils.js]: removed again as not needed for now

**Pull request merged**